salt/core/network/files/netplan_linode.yaml

```diff
@@ -0,0 +1,23 @@
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    eth0:
+      {#- Disable auto configuration so we can statically compile. #}
+      dhcp4: no
+      dhcp6: no
+      accept-ra: no
+      addresses:
+	{#- networkd makes the latest IP the primary IP so order matters here #}
```

Comment:
I assume "latest" means "last"? I changed it to avoid ambiguity.

That said, https://www.linode.com/docs/guides/linux-static-ip-configuration/#ubuntu-1804-and-2004 puts the SLAAC address last. It also includes a private IP. I don't know if "IPv6 – Link Local" in Linode matches that private IP or is something else.

Reply:
> I assume "latest" means "last"? I changed it
✔✔

In my testing I've had *lots* of issues due to the ordering the IPv6 IPs. The config in this PR is the most reliable set up. 
As I remember, putting SLAAC last will prioritise the other IPv6 address (from the Linode pool) over our IPv6 pool.
The server picks up the Link Local address automatically so no need for manual configuration.

Reply:
I'm having trouble finding documentation about the priority order. This thread suggests that the first entry is prioritized, not the last: https://bugs.launchpad.net/netplan/+bug/1766656

The manpage for systemd-network doesn't seem to describe any ordering logic: https://manpages.ubuntu.com/manpages/bionic/man5/systemd.network.5.html

Reply:
Anyhow, the logic seems to be undocumented/unspecified, so if this config works, let's go ahead with the merge. I've added comments to the file, in case we later discover documented logic.

Reply:
Agreed - the documentation around Netplan is poor.
I'm happy that this configuration is working fine on OCP14.
I will merge this PR tomorrow morning (20 Apr 2022). :)